README.md

```diff
@@ -1,3 +1,191 @@
-# React-Bookstore

```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍 Just like I have made my logo which you can see in the image above ⬆️